.travis.yml

```diff
@@ -10,3 +10,10 @@ before_script:
 
```

Comment:
I know it's not part of the scope of this ticket, but I hear and see a lot of issues with Travis lately. Is it worth a shot moving to CircleCI?

Reply:
Let's see how reliable Travis continues to be and make a move if it is a deal breaker for us